plexupdate.sh

```diff
@@ -119,21 +118,16 @@ else
 	fi
 fi
 
-if [ ! "${RELEASE}" = "" ]; then
-	echo "ERROR: RELEASE keyword is deprecated and should be removed from .plexupdate" >&2
-	echo "       Use DISTRO and BUILD instead to manually select what to install (check README.md)" >&2
-	exit 255
-fi
-
```

Comment:
This wasn't removed, it was just [moved further down](https://github.com/mrworf/plexupdate/pull/88/files#diff-7ac4485a9cb6304fa453637bd2eff8c0R201) so that all of our config-related ERROR messages are in the same spot.